channels_redis/core.py

```diff
@@ -81,9 +82,15 @@ def decode_hosts(self, hosts):
         # Decode each hosts entry into a kwargs dict
         result = []
         for entry in hosts:
-            result.append({
-                "address": entry,
-            })
+            if len(entry) == 3:
```

Comment:
You don't need this alternative format - use Redis URIs instead, like `redis://host:6379/0`

Reply:
Yup will do, was following the format of `TEST_HOSTS` but redis urls make  more sense